### ISSUE TITLE
Re-enable parser unit tests

### ISSUE THREAD

**Issue by vlada-shubina** [dotnet/templating#4965]
The following unit tests are disabled due to open issues in https://github.com/dotnet/command-line-api

https://github.com/dotnet/command-line-api/issues/1474
InstantiateTests.CanEvaluateInvalidParameters (1 data set)

https://github.com/dotnet/command-line-api/issues/1503
ListTests.List_CanParseColumns (3 data sets)
SearchTests.List_CanParseColumns (3 data sets)

https://github.com/dotnet/command-line-api/issues/1495
TabCompletionTests.Install_GetSuggestionsAfterInteractive
TabCompletionTests.Install_GetSuggestionsAfterOptionWithArg

https://github.com/dotnet/command-line-api/issues/1496
TabCompletionTests.Instantiate_CanSuggestLanguages

They should be reenabled once corresponding issues are resolved

**Comment by marcpopMSFT** [comment]
This issue is being closed as won't fix. It has been open for an extended period without maintainer activity or community demand, and is unlikely to be addressed.

The dotnet/templating repository has been consolidated into [dotnet/sdk](https://github.com/dotnet/sdk). If this issue is still important to you, please open a new issue in [dotnet/sdk](https://github.com/dotnet/sdk/issues/new) with updated context so the team can re-evaluate.

<details>
<summary>Triage details</summary>

**Summary:** Internal tracking issue to re-enable parser unit tests disabled due to command-line-api bugs

**Reasoning:** This is a nearly 4-year-old internal tracking issue about re-enabling disabled unit tests that were blocked by command-line-api issues. It has zero comments, zero upvotes, and no activity since creation. With the consolidation into dotnet/sdk, the test infrastructure and parser dependencies have likely changed significantly, making this stale tracking issue irrelevant.

</details>
